### DISCUSSION TITLE
群晖挂载成功 但是Plex扫描文件卡死

### DISCUSSION THREAD

**Discussion by lz-guo** [messense/aliyundrive-webdav#869] [General]
首先感谢大佬开发这个项目 给爱看电影的大家提供了很多方便 
之前在Win电脑上用docker挂载webdav 再用Raidrive映射成本地磁盘后用Apple TV的Plex观看 一切非常顺利

最近买了NAS（DS220+）同样成功挂载 也在File Station建立了远程连接 网页浏览器也可以正常看到文件 但是一旦在Plex连接媒体文件夹 就会发现扫描卡死的情况 不清楚这个问题是Plex导致的还是webdav导致的

初步debug感觉跟阿里云盘的文件夹名字含有中文或特殊符号有关系

一些观察到的现象：

1. Plex自动找到的Webdav路径是：/var/tmp/user/1026/gvfs/dav:host=192.168.XX.XX,port=XXXX,ssl=false,user=XXXX/Media/Films
（这个挂载路径是否正确呢？）
2. Plex可以通过以上路径正常扫描到一些电影文件夹而不会卡死（目前测试通过的两个文件夹名字都不含中文）
3. 卡死问题出现在这个文件夹：/Media/Films/青春变形记2022
4. Plex添加文件夹时 如果选择了这个文件夹 里面的内容似乎是跳转到了root。猜想是这个循环的路径导致了Plex卡死。
<img width="421" alt="image" src="https://github.com/messense/aliyundrive-webdav/assets/62634252/0a643e0c-23b5-4768-a169-66efcbd7f1e9">

5. 但是浏览器可以正常看到这个文件夹的内容
<img width="625" alt="image" src="https://github.com/messense/aliyundrive-webdav/assets/62634252/29a571f0-d6b1-4121-b034-6c0cd049b833">

6. 有时候ssh到以上路径 也会出现ls不出文件列表的情况 但是不能稳定复现

写了一大堆😅 不知道有没有小伙伴也遇到相同的问题。。

**Comment by i-iooi-i** [comment]
也是遇到了相同的问题，有大神找到解决方案吗？
